crates/js-sys/src/lib.rs

```diff
@@ -3522,6 +3522,37 @@ impl JsString {
             None
         }
     }
+
+    /// Returns whether this string is a valid UTF-16 string.
+    ///
+    /// This is useful for learning whether `String::from(..)` will return a
+    /// lossless representation of the JS string. If this string contains
+    /// unpaired surrogates then `String::from` will succeed but it will be a
+    /// lossy representation of the JS string because unpaired surrogates will
+    /// become replacement characters.
+    ///
+    /// If this function returns `false` then to get a lossless representation
+    /// of the string you'll need to manually use `iter` method (or
```

Comment:
Nit: manually use `iter` -> manually use the `iter`
Nit: or `char_code_at` -> or the `char_code_at`
Nit: the raw code points -> the raw character codes

(["code point" refers to a complete character](https://en.wikipedia.org/wiki/Character_encoding#Terminology), not a specific encoding, so it cannot refer to a byte sequence)